zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -106,6 +106,49 @@ pub(crate) const N_BYTES_GAS: usize = N_BYTES_U64;
 // Number of bytes that will be used for call data's size.
 pub(crate) const N_BYTES_CALLDATASIZE: usize = N_BYTES_U64;
 
+// Number of bytes that will be used for block values
+pub(crate) const N_BYTES_COINBASE: usize = N_BYTES_ACCOUNT_ADDRESS;
+pub(crate) const N_BYTES_GAS_LIMIT: usize = N_BYTES_U64;
+pub(crate) const N_BYTES_NUMBER: usize = N_BYTES_U64;
+pub(crate) const N_BYTES_TIMESTAMP: usize = N_BYTES_U64;
+pub(crate) const N_BYTES_DIFFICULTY: usize = N_BYTES_WORD;
+pub(crate) const N_BYTES_BASE_FEE: usize = N_BYTES_WORD;
+pub(crate) const N_BYTES_CHAIN_ID: usize = N_BYTES_U64;
+pub(crate) const N_BYTES_PREV_HASH: usize = 256 * N_BYTES_WORD;
+
+pub(crate) const N_BYTES_BLOCK: usize = N_BYTES_COINBASE
+    + N_BYTES_GAS_LIMIT
+    + N_BYTES_NUMBER
+    + N_BYTES_TIMESTAMP
+    + N_BYTES_DIFFICULTY
+    + N_BYTES_BASE_FEE
+    + N_BYTES_CHAIN_ID
+    + N_BYTES_PREV_HASH;
+
+pub(crate) const N_BYTES_EXTRA_VALUE: usize = N_BYTES_WORD + N_BYTES_WORD;
+
+// Number of bytes that will be used for tx values
+pub(crate) const N_BYTES_TX_NONCE: usize = N_BYTES_WORD;
```

Comment:
there are some mis-aligned with `ethers-core` the transaction.nonce/gas_limit are all U256, looks like tech-dept. I can fixed it in 2 steps 
1. fixed in this pr to treat nonce/gas_limit as u64 in pi_circuit/constant definition
2.[another issue + pr] submit another pr to modify nonce/gas_limit to u64 across whole codebase. It will touch quite amount of files
wdyt about this plan

Reply:
Have done point 1 in latest commits